src/main/java/org/opentripplanner/routing/car_park/CarPark.java

```diff
@@ -75,4 +83,9 @@ public int hashCode() {
     public String toString () {
         return String.format(Locale.US, "Car park %s at %.6f, %.6f", name, y, x);
     }
+
+    public boolean hasFewSpacesAvailable() {
+        var percentFree = ((float) spacesAvailable / maxCapacity);
+        return !(Double.isNaN(percentFree)) && percentFree < 0.1;
```

Comment:
We should have limits for absolute availability numbers as well, I think. For parkings with less than ten spaces, we should give an alert if only one space is available. More than 20 or 30 parkings IMHO is sufficient if the parking has more than 200 maxCapacity